core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/CompactEntityComponentSpec.scala

```diff
@@ -1348,18 +1348,24 @@ class CompactEntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatch
     // insert the entities
     runAndWait(q.batchCreateEntities(workspaceId, entities, allowUpsert = false)) shouldBe entities.size
     // retrieve the entities; retrieved value includes its id
-    entities.map { entity =>
+    val recs = entities.map { entity =>
       val actual = runAndWait(q.getEntity(workspaceId, entity.entityType, entity.name))
       actual should not be empty
       val rec = actual.get
-      // check entityType, name, and attributes
       rec.toEntity shouldBe entity
-      // check other db columns which are not present in the Entity object
       rec.deleted shouldBe false
       rec.recordVersion shouldBe 0
-
       rec
     }
+
+    // List all entities of the first entity type and check they match
+    if (entities.nonEmpty) {
+      val entityType = entities.head.entityType
+      val listed = runAndWait(q.listEntities(workspaceId, entityType))
+      listed.map(_.toEntity) should contain theSameElementsAs entities.filter(_.entityType == entityType)
+    }
+
+    recs
```

Comment:
I don't feel super strongly about this, but I'd prefer a dedicated test instead of adding to this existing test; in the case of any test failure it would make it clearer what went wrong.